pkg/eigenState/stakerShares/stakerShares.go

```diff
@@ -46,7 +46,7 @@ type StakerShareDeltas struct {
 	WithdrawalRootString string `gorm:"-"`
 }
 
-func NewSlotID(staker string, strategy string, strategyIndex uint64, transactionHash string, logIndex uint64) types.SlotID {
+func NewSlotID(transactionHash string, logIndex uint64, staker string, strategy string, strategyIndex uint64) types.SlotID {
```

Comment:
I think you could get away with just strategy Index right (i.e instead of strategy too)?

Reply:
In theory yes, but I know for a fact this will be 100% unique. I already encountered some funky edge cases, so gonna leave it like this.